src/Nox.Types.EntityFramework/EntityBuilderAdapter/OwnedNavigationBuilderAdapter.cs

```diff
@@ -26,6 +26,11 @@ public ReferenceNavigationBuilder HasOne(string relatedTypeName, string? navigat
         {
             return OwnedNavigationBuilder!.HasOne(relatedTypeName, navigationName!);
         }
+        public CollectionNavigationBuilder HasMany(string navigationName)
+        {
+            // TODO: fix owned entity many to many realtionships
+            throw new Exception("HasMany is not supported for Owned navigation. Will be implemented later.");
```

Comment:
Rather to instantiate NotImplementedException

Reply:
fixed